Jenkinsfile

```diff
@@ -76,13 +76,28 @@ def cleanup_master() {
 	}
 }
 
+def archive_logs() {
+	sh """
```

Comment:
If you use `sh '''` (single quotes) you do not need to perform any escaping for groovy.